perf-regression/theta-gpu/gpu-qsub

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+#COBALT -A radix-io
+#COBALT -n 2
+#COBALT -t 60
+#COBALT --mode script
+#COBALT -M vchoi@hdfgroup.org
```

Comment:
We may want to think about who this should ultimately be directed to. We may even want to not specify this option by default

Reply:
I think I will make that into a a comment about  using -M option for sending mail.